docs/en/tutorial.md

```diff
@@ -2,29 +2,25 @@
 slug: /en/tutorial
 sidebar_label: Advanced Tutorial
 sidebar_position: 0.5
-keywords: [clickhouse, install, tutorial, dictionary, dictionaries]
+keywords: [clickhouse, install, tutorial, dictionary, dictionaries, example, advanced, taxi, new york, nyc]
 ---
-import SQLConsoleDetail from '@site/docs/en/_snippets/_launch_sql_console.md';
-
 # Advanced Tutorial
 
-## What to Expect from This Tutorial?
-
-In this tutorial, you will create a table and insert a large dataset (two million rows of the [New York taxi data](/docs/en/getting-started/example-datasets/nyc-taxi.md)). Then you will run queries on the dataset, including an example of how to create a dictionary and use it to perform a JOIN.
+## Overview
 
-:::note
-This tutorial assumes you have access to a running ClickHouse service.  If not, check out the [Quick Start](./quick-start.mdx).
-:::
+Learn how to ingest and query data in ClickHouse using a New York City taxi example dataset. 
 
-## 1. Create a New Table
+### Prerequisites
+You need access to a running ClickHouse service to complete this tutorial. For instructions, see the [Quick Start](./quick-start.mdx) guide.
 
-The New York City taxi data contains the details of millions of taxi rides, with columns like pickup and drop-off times and locations, cost, tip amount, tolls, payment type and so on. Let's create a table to store this data...
+## Create a new table
 
-1. Connect to the SQL console
+The New York City taxi dataset contains details about millions of taxi rides, with columns including tip amount, tolls, payment type, and more. Create a table to store this data.
 
-  <SQLConsoleDetail />
```

Comment:
These two giant screenshots seem extremely unnecessary and disruptive to the reader's experience.